pkg/models/node_info.go

```diff
@@ -63,28 +55,57 @@ func (p mergeProvider) GetLabels(ctx context.Context) map[string]string {
 	return labels
 }
 
+type LabelsProvider interface {
+	GetLabels(ctx context.Context) map[string]string
+}
+
+type mergeProvider struct {
+	providers []LabelsProvider
+}
+
 func MergeLabelsInOrder(providers ...LabelsProvider) LabelsProvider {
 	return mergeProvider{providers: providers}
 }
 
+// NodeInfo godoc
+//
+// @ID			NodeInfo
+// @Summary		Information about the node.
+// @Description	Information about the node.
+// @Tags		Ops
+// @Produce		json
+// @Success	200	{object}	NodeInfo
+// @Failure	500	{object}	string
+// @Router		/api/v1/agent/node [get]
```

Comment:
Swagger comments should be added on the actual method which serves the request, rather than the data structure, no?

Reply:
Swagger needs the godoc but you're right (I think) no router here